dags/common/load.py

```
@@ -0,0 +1,66 @@
+    # 기존 데이터 삭제
+    city_ids = tuple(df['city_id'].unique())
+    city_ids_str = f"({city_ids[0]})" if len(city_ids) == 1 else str(city_ids)
+    
+    min_dt = df['dt'].min()
+    max_dt = df['dt'].max()
+
+    delete_sql = f"""
+        DELETE FROM mart.weather 
+        WHERE city_id IN {city_ids_str} 
+        AND dt BETWEEN '{min_dt}' AND '{max_dt}';
+    """
```

Ruff (0.14.10)

Possible SQL injection vector through string-based query construction

(S608)